sections/basics/react-native.md

```diff
@@ -51,6 +51,9 @@ Some of the differences to the web-version are, that you cannot use the
 keyframes or global styles. We will also warn you if you use media queries or
 nest your CSS.
 
+> Note that React Native does not support Styling Children from the Parent
```

Comment:
Good start, I'd word it more like this:

Another difference is React Native does not support cascading styles, so when you write CSS for a React Native component it essentially becomes an inline style. The implication here is typical CSS selectors don't do anything, e.g. `* {}`, `> Text`, etc. It's recommended to make use of `ThemeProvider` to pass styles down to overcome this limitation.

Reply:
That's well worded as long as the implication is also defined in the paragraph as beginner developers may not be aware of the difference.

I think a provider is an interesting approach. If we want to suggest that I would like to see an example as to ensure developers are creating re-usable styles instead of implementing it incorrectly.